docs/src/modules/components/AppFrame.js

```diff
@@ -192,6 +199,9 @@ class AppFrame extends React.Component {
               <Notifications />
               <MarkdownLinks />
               <AppBar className={appBarClassName}>
+                <Typography className={classes.banner}>
+                  {`⚠️ This is an alpha version (v${process.env.LIB_VERSION})`}
```

Comment:
Since the rest of the `next` homepage is localized, could we localize this too?

Also, how about: 

'⚠️ This is the documentation for the `next` release of Material-UI. Currently: {{version}}.'

(Backticks only as a form of punctuation, if we wanted actual italics we'd have to use an interpolation and pass in '<em>next</em>'...)

Reply:
Ok, I'm translating, but I think that your suggestion is too long. People don't read.

Reply:
It's 6 more words. If you drop "This is the", 3 more words, and drop "of Material-UI", 1 more word. If developers can't read 8 (or 11) words, they aren't going to get very far in the docs!

> This is an alpha version

It isn't an alpha version though... It's the docs for an an alpha version. And since it will change from an alpha, to a beta, to a pre-release (?), simpler to refer to it as the docs for the `next` release. No need to change the original (I guaranteed it will be forgotten), or to update the translations.